src/Shikensu.hs

```diff
@@ -122,21 +122,21 @@ makeDefinition :: FilePath -> Pattern -> FilePath -> Definition
 makeDefinition _rootDirname _pattern _workspacePath =
     let
         workingDir  = cleanPath . (commonDirectory) $ _pattern
-        localPath   = cleanPath . (stripPrefix workingDir) . cleanPath $ _workspacePath
+        localPath'  = cleanPath . (stripPrefix workingDir) . cleanPath $ _workspacePath
```

Comment:
Thanks! 😄 🎉 

Sidenote: Not a super big fan of the `'` suffix, I think we should rename it to something else instead. Thoughts?